tests/fixtures/structural/test-001/POSCAR

```diff
@@ -0,0 +1,14 @@
+Silicon FCC
```

Comment:
Methane??

Reply:
Yes, methane. I will fix the header line. I thought that methane would be a good small molecule test.